Resources/Prototypes/Entities/Clothing/Head/eva-helmets.yml

```diff
@@ -81,12 +81,22 @@
       - state: equipped-head-light
       head-vox:
       - state: equipped-head-light-vox
+      # Begin DeltaV Additions - add modified vulp helmet
+      head-vulpkanin:
```

Comment:
just have 1 comment on the head-vulpkanin state since its the only one added
instead of begin/end